doc/source/whatsnew/v2.1.2.rst

```diff
@@ -17,6 +17,7 @@ Fixed regressions
 - Fixed regression in :meth:`DataFrame.join` where result has missing values and dtype is arrow backed string (:issue:`55348`)
 - Fixed regression in :meth:`DataFrame.resample` which was extrapolating back to ``origin`` when ``origin`` was outside its bounds (:issue:`55064`)
 - Fixed regression in :meth:`DataFrame.sort_index` which was not sorting correctly when the index was a sliced :class:`MultiIndex` (:issue:`55379`)
+- Fixed performance regression in Copy-on-Write mechanism (:issue:`55256`, :issue:`55245`)
```

Comment:
```suggestion
- Fixed performance regression with wide DataFrames, typically methods where all columns were accessed individually (:issue:`55256`, :issue:`55245`)
```

One attempt to make it not CoW specific and still somewhat concrete

Reply:
Thx I like that version